doc/source/examples/resolved-cfd-dem/sedimentation-1-particle/sedimentation-1-particle.rst

```diff
@@ -0,0 +1,345 @@
+
+* The ``grid arguments`` is set to  ``5,8,5: 0,0,0 : 10 , 16 ,10 : true``. This section has 3 subsections. First ``5,8,5`` describes the initial subdivision of the box. This subdivision has been chosen as it is the smallest mesh we can do of the box in order to have cubic elements. Secondly ``0,0,0 : 10 , 16 ,10`` describes the 2 points from which we have derived the rectangular box (0,0,0) and  (10,16,10). Finally, we have ``true``, which is a boolean to activate the coloration of the boundary. This will allow us to define a boundary condition separately for each side of the box.
+
+* The ``initial refinement`` is set to 1. This will ensure to have a base mesh that is a bit smaller than the particle.
+
+
+Mesh Adaptation Control
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+.. code-block:: text
+
+	subsection mesh adaptation
+	   # Fraction of coarsened elements
+	  set fraction coarsening  = 0.3
+
+	  # Fraction of refined elements
+	  set fraction refinement  = 0.05
+
+	  # How the fraction of refinement/coarsening are interepretedChoices are
+	  # <number|fraction>.
+	  set fraction type        = number
+
+	  # Frequency of the mesh refinement
+	  set frequency            = 1
+
+	  # Maximum number of elements
+	  set max number elements  = 750000
+
+	  # Maximum refinement level
+	  set max refinement level = 6
+	 # minimum refinement level
+	  set min refinement level =0
+
+	  # Type of mesh adaptationChoices are <none|uniform|kelly>.
+	  set type                 = kelly
+
+	  # Variable for kelly estimationChoices are <velocity|pressure>.
+	  set variable             = velocity
+	end
+
+* The ``fraction coarsening`` is set to 0.3. This limits the accumulation of elements when the particle is moving. It allows for cells far from the particle to be coarsened when the particles get further away.
+
+* The ``fraction refinement`` is set to 0.05. The objective here is to refine elements that become close to the particle when it's moving. This will mostly refine elements around the particle that are not included in the refinement zone around the particle. The refinement zone around the particle will be discussed in more detail in the IB particle section.
+
+* The ``set frequency`` is set to 1. Since the particle is moving at each time step, the refinement zone around it should be reevaluated at each time step.
+
+* The ``max refinement level`` is set to 6. This parameter limits how small the elements around the particle can get limiting the total number of elements in the problem. Here we limit the mesh size to 48 elements per diameter of the particle. This should be sufficient to get accurate results.
+
+* The ``type`` is set to ``kelly``. Since the particle is moving and we don't want a uniform refinement of all the cell, we use the kelly error estimator based on the ``velocity`` variable.
```

Comment:
```suggestion
* The ``type`` is set to ``kelly``. Since the particle is moving and we do not want a uniform refinement of all the cells, we use the kelly error estimator based on the ``velocity`` variable.
```